### PR TITLE
EFGS: Use symptom date in VC

### PR DESCRIPTION
**Pull request by jendakol**
When VC with no symptom date is used, it'S later ignored and doesn't trigger "risk encounter".

### CODE REVIEW

File: internal/functions/efgs/api/structs.go [hunk 1]
```diff
@@ -16,8 +16,10 @@ type GenericVerServerResponse struct {
 
 //IssueCodeRequest Issue code request to the Verification server
 type IssueCodeRequest struct {
-	Phone    string `json:"phone"`
-	TestType string `json:"testType"`
+	Phone       string `json:"phone"`
+	TestType    string `json:"testType"`
+	SymptomDate string `json:"symptomDate"` // ISO 8601 formatted date, YYYY-MM-DD
+	TestDate    string `json:"testDate"`
```
Comment (marekaf) [inline internal/functions/efgs/api/structs.go]: why is this boilerplate here and we are not using the vf server struct / "API"? https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/api/api.go#L168

Comment (jendakol) [reply internal/functions/efgs/api/structs.go]: That's a very... ugly and right question :neutral_face:

**Comment by marekaf** [comment]
there's a conflict, rebase pls :D

**Comment by jendakol** [comment]
I know, I will rebase it once you approve the PR. I didn't want to force-push the branch now so you can see the diff ;-)

**Review by marekaf — APPROVED** [review]
lgtm

**Pull request merged**